lib/rest/authy/v1/service.d.ts

```diff
@@ -254,17 +293,19 @@ declare class ServiceInstance extends SerializableClass {
   /**
    * fetch a ServiceInstance
    *
+   * @param opts - Options for request
    * @param callback - Callback to handle processed record
    */
-  fetch(callback?: (error: Error | null, items: ServiceInstance) => any): Promise<ServiceInstance>;
+  fetch(opts?: ServiceInstanceFetchOptions, callback?: (error: Error | null, items: ServiceInstance) => any): Promise<ServiceInstance>;
   friendlyName: string;
   links: string;
   /**
    * remove a ServiceInstance
    *
+   * @param opts - Options for request
    * @param callback - Callback to handle processed record
    */
-  remove(callback?: (error: Error | null, items: ServiceInstance) => any): Promise<boolean>;
+  remove(opts?: ServiceInstanceDeleteOptions, callback?: (error: Error | null, items: ServiceInstance) => any): Promise<boolean>;
```

Comment:
Need to mark the overall change as `breaking` so that it rolls the minor version.

Reply:
Noted. Where would we do this for breaking library changes since PRs are automatically added to the changelog?

Reply:
Just stick the word `breaking` in the PR title somewhere.